joern-cli/frontends/csharpsrc2cpg/src/main/scala/io/joern/csharpsrc2cpg/astcreation/AstForExpressionsCreator.scala

```diff
@@ -454,4 +455,40 @@ trait AstForExpressionsCreator(implicit withSchemaValidation: ValidationMode) {
     )
   }
 
+  private def astForConditionalAccessExpression(condAccExpr: DotNetNodeInfo): Seq[Ast] = {
+    val baseNode = createDotNetNodeInfo(condAccExpr.json(ParserKeys.Expression))
+    val baseAst  = astForNode(baseNode)
+
+    val fieldIdentifier = NewFieldIdentifier()
+      .code(baseNode.code)
+      .canonicalName(baseNode.code)
+      .lineNumber(condAccExpr.lineNumber)
+      .columnNumber(condAccExpr.columnNumber)
+
+    val baseTypeFullName = getTypeFullNameFromAstNode(baseAst)
+
+    Try(createDotNetNodeInfo(condAccExpr.json(ParserKeys.WhenNotNull)(ParserKeys.Name))).toOption match {
+      case Some(node) =>
+        // Got a member access
+        val typ = scope
+          .tryResolveFieldAccess(s"${node.code}", Option(baseTypeFullName))
+          .map(_.typeName)
+          .getOrElse(Defines.Any)
```

Comment:
If this becomes the following, then you don't need to re-do this logic on `newOperatorCallNode` line 483
```suggestion
          .orElse(Option(Defines.Any))
```